Dockerfile

```diff
@@ -0,0 +1,7 @@
+from node
+RUN mkdir /app
+WORKDIR /app
+COPY [ "package.json", "src", "public", "/app/"]
+RUN yarn
+EXPOSE  3000
```

Comment:
Not sure if this is a good idea or not: if we always expose port 80 for web servers, it'll make deployment slightly easier across projects. It does make things slightly more confusing as mentioned in cityofaustin/joplin#7

Reply:
Sorry, I don't follow what you're suggesting here. Should line 6 be `EXPOSE 80` instead?

Reply:
Yes that's what i'm suggesting